tool/src/org/antlr/v4/parse/ANTLRParser.g

```diff
@@ -60,36 +60,20 @@ options {
 // specify any such tokens
 tokens {
     RULE;
-	PREC_RULE; // flip to this if we find that it's left-recursive
     RULES;
     RULEMODIFIERS;
-    RULEACTIONS;
     BLOCK;
     OPTIONAL;
     CLOSURE;
     POSITIVE_CLOSURE;
     RANGE;
     SET;
-    CHAR_RANGE;
     EPSILON;
     ALT;
-    ALTLIST;
     ID;
-    ARG;
```

Comment:
Wow. We were incredibly lazy to leave this grammar the way it was ha ha! thanks!